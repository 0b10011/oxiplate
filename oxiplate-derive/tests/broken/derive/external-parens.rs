use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate("foo.oxip")]
struct Data;

fn main() {}
