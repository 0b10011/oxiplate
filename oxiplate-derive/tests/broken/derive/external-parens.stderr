error: proc-macro derive panicked
 --> tests/broken/derive/external-parens.rs:3:10
  |
3 | #[derive(Oxiplate)]
  |          ^^^^^^^^
  |
  = help: message: not yet implemented: need to handle when non-name-value data is provided
